website/content/en/docs/building-operators/golang/webhook.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Admission Webhooks
+linkTitle: Webhook
+weight: 30
+description: An in-depth walkthough of admission webhooks.
+---
+
+## Create a validating or mutating Admission Webhook 
+
+Admission webhooks are HTTP callbacks that receive admission requests and do something with them. It is registered with Kubernetes, and
+will be called by Kubernetes to validate or mutate a resource before being stored. There are two types of admission webhooks.
+
+#### 1. Validating admission webhook
+
+Validating webhooks can be used to perform validations that go beyond the capabilities of OpenAPI schema validation, 
+such as ensuring a field is immutable after creation or higher level permissions checks based on the user that is making 
+the request to the API server. It can reject the request, but it cannot modify the object that they are receiving in the request.
+
+#### 2. Mutating admission webhook
+
+Mutating webhooks are most frequently used for defaulting, by adding default values for unset fields in the resource on creation. 
+They can modify objects by creating a patch that will be sent back in the admission response.
+
+For more background on Admission webhooks, refer to the [Kubebuilder documentation](https://book.kubebuilder.io/reference/admission-webhook.html) or the [official Kubernetes documentation](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) on the topic. 
+You can also refer to the [Kubebuilder webhook walkthrough](https://book.kubebuilder.io/cronjob-tutorial/webhook-implementation.html), which is similar in content to this guide. 
+Kubebuilder also has a guide that walks through implementing webhooks for their example `CronJob` resource.
+
+### Create Validation Webhook
+
+As an example, let's start by creating a validation webhook.
+
```

Comment:
As an example, let's walk through the scaffolding of a validation webhook for the sample memcached operator.

Reply:
Updated.